scripts/nbconverted/adage_training.py

```diff
@@ -178,8 +178,11 @@
 # In[17]:
 
 # Output weight matrix of gene contributions per node
-weight_matrix = pd.DataFrame(autoencoder.get_weights()[0], index=rnaseq_df.columns)
-weight_file = os.path.join('data', 'adage_gene_weights.tsv')
+weight_file = os.path.join('results', 'adage_gene_weights.tsv')
+
+weight_matrix = pd.DataFrame(autoencoder.get_weights()[0], index=rnaseq_df.columns,
+                             columns=range(1, 101)).T
```

Comment:
I'm assuming this is just a move away from zero-based indexing?

Reply:
correct - to maintain consistency...for now. I intend to update all of these at a later date (as per #86)